src/Rules.Framework/Providers/InMemory/DataModel/RuleDataModel.cs

```diff
@@ -13,7 +14,6 @@ internal sealed class RuleDataModel<TContentType, TConditionType>
         public DateTime? DateEnd { get; set; }
 
         public string Name { get; set; }
-
```

Comment:
don't remove the space please (or remove all :) )

Reply:
Added back